lib/workload/stateful/securitygroup/component.ts

```diff
@@ -2,21 +2,21 @@ import { Construct } from 'constructs';
 import { IVpc, SecurityGroup } from 'aws-cdk-lib/aws-ec2';
 import { aws_ec2 } from 'aws-cdk-lib';
 
-export interface Props {
-  vpc: IVpc;
+export interface SecurityGroupProps {
+  securityGroupName: string,
+  securityGroupDescription: string,
 }
 
 export class SecurityGroupConstruct extends Construct {
-  public static readonly ORCABUS_LAMBDA_SECURITY_GROUP: string = 'OrcaBusLambdaSecurityGroup';
 
-  constructor(scope: Construct, id: string, props: Props) {
+  constructor(scope: Construct, id: string, vpc: IVpc, props: SecurityGroupProps) {
     super(scope, id);
 
-    const lambdaSecurityGroup = new SecurityGroup(this, 'LambdaSecurityGroup', {
-      securityGroupName: SecurityGroupConstruct.ORCABUS_LAMBDA_SECURITY_GROUP,
-      vpc: props.vpc,
+    const lambdaSecurityGroup = new SecurityGroup(this, id + 'LambdaSecurityGroup', {
```

Comment:
Ditto `OrcaBusLambdaSecurityGroup`?

Reply:
Yup. Same explanation. The second argument is for Construct ID. 

Referencing to stateful security group name is happening in next line, as part of `props`.